<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgDocAdd</class>
 <widget class="QDialog" name="DlgDocAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="HLayoutLists">
     <item>
      <widget class="QListWidget" name="ListUnlinked">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="VLayoutArrows">
       <item>
        <widget class="QToolButton" name="ButtonLink">
         <property name="text">
          <string>...</string>
         </property>
         <property name="arrowType">
          <enum>Qt::RightArrow</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="ButtonUnlink">
         <property name="text">
          <string>...</string>
         </property>
         <property name="arrowType">
          <enum>Qt::LeftArrow</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="ListLinked">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Right-click the documents in the lists for more options</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="HLayoutButtons">
     <item>
      <widget class="QPushButton" name="ButtonOpenFileDirectory">
       <property name="text">
        <string>Open files directory</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="HSpacerButtons">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonOK">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
